Assets/Src/AuthController/REST/REST Response DTOs/GoogleIdTokenResponse.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Src.AuthController.REST.REST_Response_DTOs
+{
+    [Serializable]
+    public class GoogleIdTokenResponse
+    {
+        [JsonProperty("access_token")]
+        public string accessToken;
+        
+        [JsonProperty("expires_in")]
+        //In seconds
```

Comment:
To highlight such information (which is important information in fact) you should either use docstrings or change name of the class member to more clear one.